papers/vyas_ramasubramani/94_vyas_ramasubramani.rst

```diff
@@ -348,28 +341,33 @@ As a result, our initial comparisons will focus on existing packages that solve
+However, maintaining a server for workflow management can be cumbersome, and it introduces additional unnecessary complexities.
+
+A more significant limitation of these other tools is that their data representations are closely tied to the workflow execution, making it much more challenging to access the data outside the context of the workflow.
+Concretely, these software typically store data in a specific location based on a particular instance of an operation's execution, so the data can only be found by looking for that specific instance of the operation.
+Conversely, in ``signac`` the data is identified by its own metadata, namely its state point, so once it has been generated its access is no longer linked to a specific instance of a ``signac-flow`` operation (assuming that ``signac-flow`` is being used).
+
+Of course, knowing exactly where and how data was generated and transformed, *i.e.*, the data provenance, is also valuable information.
+Two tools that are specialized for this task are Sacred and Sumatra.
+Superficially, the ``signac`` framework appears especially similar to Sacred.
+Both use decorators to convert functions into executable operations, and configurations can be injected into these functions (in ``signac``'s case, using the job object).
+Internally, Sacred and ``signac-flow`` both depend on the registration of particular functions with some internal API: in ``signac-flow``, functions are stored as operations within the ``FlowProject``, whereas Sacred tracks functions through the *Experiment* class.
+However, the focus of Sacred is not to store data or execute workflows, but instead to track when an operation was executed, the configuration used, and whether any data was saved.
 Therefore, in principle ``signac`` and Sacred are complementary pieces of software that could be used in concert to achieve different benefits.
 
-.. TODO any plans to integrate the configuration/use of Sacred into signac? It sounds like any signac user could be encouraged to improve his/her provenance by supporting Sacred in signac? ... Well, nvm, you answered it in the next paragraph :)
+We have found that integrating Sacred with ``signac`` is in fact quite simple.
+Once functions are registered with either a Sacred Experiment or a ``signac-flow`` ``FlowProject``, the operations can be run either through Python or on the command line.
+While both tools typically advocate using their command line interfaces, the two can be integrated by using one from the command line while having it internally call the other through the other's Python interface.
+Since ``signac-flow`` provides a more extensive set of command line options that are likely to be regularly changed, we advocate using the ``signac-flow`` command line functionality and encoding a Sacred Experiment within a ``signac-flow`` operation.
```

Comment:
Regardless of which CLI would be preferable as the "primary entrypoint" for joint Sacred and signac operation, in the end, it boils down to which project has the bandwidth and priority to get started on this first :)